src/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -64,9 +66,14 @@
     @Option(shortName=StandardOptionDefinitions.INPUT_SHORT_NAME, doc = "Input SAM or BAM file.")
     public File INPUT;
 
-    @Option(shortName=StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Base of output files to write.")
+    @Option(shortName=StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "If SUMMARY_OUTPUT is null, the base of output files to write, " +
+            "otherwise the full path to the detailed metrics.")
```

Comment:
I think that the term "full path" is confusing here as some-one might give a relative path and it will be used as such.